pkg/apis/monitoring/v1/types.go

```diff
@@ -1250,6 +1250,9 @@ type ServiceMonitorSpec struct {
 	// Per-scrape limit on length of labels value that will be accepted for a sample.
 	// Only valid in Prometheus versions 2.27.0 and newer.
 	LabelValueLengthLimit uint64 `json:"labelValueLengthLimit,omitempty"`
+	// Attaches node metadata to discovered targets. Only valid for role: pod and service.
+	// Only valid in Prometheus versions 2.35.0 and newer.
```

Comment:
I think that v2.37.0 at least is required.
```suggestion
	// Requires Prometheus v2.37.0 and above.
```